### DISCUSSION TITLE
why block cryptosites

### DISCUSSION THREAD

**Discussion by lenusch** [FiltersHeroes/KADhosts#58] [General]
Hi, 

Why blocking all `dascoin` Sites
`dascoin`, `daswallet`, `das33.io` and so on...
on PORN List?!

I took an hour to find this today.

**Comment by azet12** [comment]
@lenusch Sorry for the delayed answer. There must have been some misunderstanding on your side, because this is not a porn hosts file. It's a scam list.

**Comment by lenusch** [comment]
Whoops - then is this a proven Scam (DASC) or personal Opinion? 
Because DASC was launched this year on several markets and everything they bringt out is far more i ever seen from some other "Start ups" cryptos? Their System and Cooperation with Ledger works flawless. I am using it nearly weekly without Problems. 
Just saying ...

**Comment by azet12** [comment]
This list is based on opinions from me and some other people. To be honest with you, I don't know much about this topic. A person that reported those websites to me claimed it's a Ponzi scheme, and this list blocks them by design.

**Comment by dnmTX** [comment]
@azet12  would advise to do some research on your own before blocking domain that might or might not be what some "third party" reported, based purely on their personal opinion. Just saying, not really trying to tell you ho to manage your repo.
Cheers.

**Comment by hawkeye116477** [comment]
* [`https://subiektywnieofinansach.pl/dascoin-eldorado-ktore-konczy-sie-smetnie-monety-prawie-bez-wartosci-prokuratura-na-tropie-licencja-na-zarabianie-nie-wypalila/`](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://subiektywnieofinansach.pl/dascoin-eldorado-ktore-konczy-sie-smetnie-monety-prawie-bez-wartosci-prokuratura-na-tropie-licencja-na-zarabianie-nie-wypalila/)

* [`https://businessinsider.com.pl/finanse/kryptowaluty/dascoin-liderzy-o-przesluchaniach-przez-cbs-i-prokurature/ep3hhg3`](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://businessinsider.com.pl/finanse/kryptowaluty/dascoin-liderzy-o-przesluchaniach-przez-cbs-i-prokurature/ep3hhg3)

* [`https://www.fxmag.pl/artykul/dascoin-oficjalnie-uznany-za-scam-przez-amerykanskiego-regulatora`](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://www.fxmag.pl/artykul/dascoin-oficjalnie-uznany-za-scam-przez-amerykanskiego-regulatora)

* [`https://www.uokik.gov.pl/aktualnosci.php?news_id=15005`](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://www.uokik.gov.pl/aktualnosci.php?news_id=15005)

It was officially recognized as scam by NASAA, even Polish Office of Competition and Consumer Protection warned about it and Polish prosecutor’s office initiated an investigation, so rather something's going on.
